packages/lib/gpcircuits/src/proto-pod-gpc.ts

```diff
@@ -346,8 +405,8 @@ export class ProtoPODGPC {
    */
   public static CIRCUIT_FAMILY: ProPODGPCCircuitDesc[] = [
     // TODO(POD-P2): Pick convenient circuit sizes for MVP.
-    ProtoPODGPC.curcuitDescForParams(1, 1, 5, 9556),
-    ProtoPODGPC.curcuitDescForParams(1, 5, 8, 19223),
-    ProtoPODGPC.curcuitDescForParams(3, 10, 8, 45276)
+    ProtoPODGPC.curcuitDescForParams(1, 1, 10, 2, 2, 5, 9556),
```

Comment:
I'd suggest making sure there's more than one value for each param included in the test cases.  So, a larger list and tuple.  The last entry is meant to be larger than the test cases to make sure all the padding works.

Reply:
Also note the costs here will need to be updated when these circuits are updated.  I copied the `cost` entry from the number of non-linear constraints in the output when building from the command-line.  I foresee that being annoying to do repeatedly, so don't bother until it's time to merge.  Another good argument for the single source of truth you suggested.